bellbot_gui/package.xml

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0"?>
 <package>
   <name>bellbot_gui</name>
-  <version>0.0.10</version>
+  <version>0.0.11</version>
```

Comment:
See comment for action server package.xml. This has to be changed to `0.0.10` again.